src/namespace.js

```diff
@@ -15,6 +15,14 @@ const setup = constructor => {
 }
 
 const get = namespace => {
+	// if tram-one is setup, this will be defined
+	const tramOneIsSetup = window['tram-space']
+
+	// otherwise, we should warn
+	if (!tramOneIsSetup) {
+		throw new Error('Tram-One: app has not started yet, but hook was called. Is it being invoked outside a component function?')
+	}
+
```

Comment:
Error to throw when `window['tram-space']` is not setup. 
This happens most often when a hook is called outside of a component function (it's called immediately when a component is imported, and thus, before start has been called, and before the `window['tram-space']` has been defined.). It previously returned the following error:
```
"Cannot read property 'tram-effect-queue' of undefined"
```
But this should be more clear to developers, and not require them to understand the inner-workers (or know how to parse this somewhat cryptic error).